packages/shared/src/lib/image.ts

```diff
@@ -11,6 +11,7 @@ export const cloudinary = {
       'https://daily-now-res.cloudinary.com/image/upload/v1665563545/public/supercharge.png',
   },
   notifications: {
+    bell: '', // TODO: update when image is uploaded to cloudinary
```

Comment:
The bell icon is a icon right? Why do you want that added to cloudinary?

Reply:
I would also prefer the icon to be decided in the frontend based on which type belongs to which icon. But from the discussion we've had, it would be ideal to use a link for the icon as well to make our notifications super flexible.
cc @idoshamun

Reply:
I think we referred only to the icons inside the notification component itself, not the bell that shows the notifications

Reply:
Yes, this is that. We will be using those icons to be populated in our data, and this one is used for the first notification which is implemented in the frontend to not create a row in our table for notifications for all users.